interface/ceed-elemrestriction.c

```diff
@@ -29,12 +29,12 @@
 /**
   @brief Permute and pad indices for a blocked restriction
 
-  @param indices    Array of shape [@a nelem, @a elemsize]. Row i holds the
-                      ordered list of the indices (into the input CeedVector)
+  @param offsets    Array of shape [@a nelem, @a elemsize]. Row i holds the
+                      ordered list of the offsets (into the input CeedVector)
                       for the unknowns corresponding to element i, where
                       0 <= i < @a nelem. All indices must be in the range
                       [0, @a nnodes).
```

Comment:
```suggestion
                      [0, @a lsize).
```

Reply:
Bundled into a larger set of related changes. Thanks.